1/Dockerfile

```diff
@@ -48,6 +48,35 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/subversion.jpi.pinned && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
+    chmod -R 777 /etc/alternatives && \
+    chmod -R 777 /var/lib/alternatives && \
+    chmod -R 777 /usr/lib/jvm && \
+    chmod 777 /usr/bin && \
+    chmod 777 /usr/lib/jvm-exports && \
+    chmod 777 /usr/share/man/man1 && \
```

Comment:
yeah i meant that we could chmod them to 775 (since they are already in the root group, there is no need to chgrp them)